app/models/form/sales/pages/buyer1_income_max_value_check.rb

```diff
@@ -0,0 +1,28 @@
+class Form::Sales::Pages::Buyer1IncomeMaxValueCheck < ::Form::Page
+  def initialize(id, hsh, subsection, check_answers_card_number:)
+    super(id, hsh, subsection)
+    @depends_on = [
+      {
+        "income1_over_soft_max?" => true,
+      },
+    ]
+    @title_text = {
+      "translation" => "soft_validations.income.over_soft_max_for_la_buyer_1",
+      "arguments" => [
+        {
+          "key" => "field_formatted_as_currency",
+          "arguments_for_key" => "income1",
+          "i18n_template" => "income",
+        },
+      ],
+    }
+    @informative_text = {}
+    @check_answers_card_number = check_answers_card_number
```

Comment:
Wouldn't this always be 1? and 2 on buyer 2?

Reply:
no - needs to be nil in some cases when the value check is displayed in other subsections